src/main/java/com/solr/clientwrapper/domain/dto/solr/SolrSchemaDTO.java

```diff
@@ -0,0 +1,17 @@
+package com.solr.clientwrapper.domain.dto.solr;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@EqualsAndHashCode
+public class SolrSchemaDTO {
+
+	String tableName;
+	String name;
+	SolrFieldDTO [] attributes;
+	
```

Comment:
A list would be better than array

Reply:
Actually, this is the way Kirti Ma'am probably communicated with Mangesh, it's his part of code. I also thought the same, but.., will consult with them.